pkg/internal/checkup/executor/executor.go

```diff
@@ -185,7 +185,7 @@ func (e Executor) clearStatsTestpmd(vmiName string) error {
 	return nil
 }
 
-func (e Executor) getStatsTestpmd(vmiName string) (map[string]int64, error) {
+func (e Executor) getStatsTestpmd(vmiName string, verbosePrintsEnabled bool) (map[string]int64, error) {
```

Comment:
Executor has this as a member field.
No need to pass it as a parameter.

Reply:
good catch! DONE